project-koku/koku
# [Cost 2170]Fix for wrong error message for settings

Fixes [COST-2170](https://issues.redhat.com/browse/COST-2170)

Changes proposed in this PR:
* Handle specific error condition where the customer has access to cost settings, but has yet to create a Source

Testing Instructions:
1. Setup [settings-frontend](https://github.com/RedHatInsights/settings-frontend) to run locally
2. Setup koku api locally or in ephemeral env
3. Use customer that has no sources configured, but has RBAC perms
4.  access settings page
http://localhost:1337/beta/settings/applications/cost-management

Expected :
![Screen Shot 2022-01-25 at 9 16 20 AM](https://user-images.githubusercontent.com/57504257/150993504-38715ee8-c2cb-4384-ac2c-5ab68ec703b6.png)


or you can ping me
---
Additional Context:
